### PR TITLE
[bcl] Add a Mono.Runtime.Tests assembly which can be used to run the JIT regression tests using nunit.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: mcs/class/Mono.Runtime.Tests/Assembly/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+using System;
```
Comment (marek-safar) [inline mcs/class/Mono.Runtime.Tests/Assembly/AssemblyInfo.cs]: You could do `NO_BUILD = 1` to avoid all this

**Comment by marek-safar** [comment]
What is the motivation behind this?

**Comment by vargaz** [comment]
Its for the sdks, to make it easier to run the JIT tests there.

**Comment by luhenry** [comment]
Can't you just reuse the `sdks/android/managed/mini-test-runner.cs`? You could put it in a common place in `sdks/`, or copy paste it.

**Comment by vargaz** [comment]
This is basically the same thing. It allows us to treat the mini regression tests the same as any BCL test, so we don't need special case code anymore.

**Pull request merged**